app/src/db/models/note.ts

```diff
@@ -30,6 +32,8 @@ export default {
     return {
       noteId: input.note_id,
       activityId: input.activity_id,
+      bringForwardDate: input.bring_forward_date?.toISOString() ?? null,
+      bringForwardState: input.bring_forward_state || '',
```

Comment:
Can we make this nullable instead of defaulting to an empty string. NULL is a better representation of no value.